ibi/coreos-installer/Makefile

```diff
@@ -14,6 +14,12 @@ COREOS_INSTALLER=$(TMPDIR)/coreos-installer
 ISO_PATH=$(ISO_DIR)/rhcos-ibi.iso
 KCLI_TEMPLATE=-P iso="$(ISO_PATH)" -P memory=33792 -P numcpus=16 -P disks=['$(KCLI_TEMPLATE_DISK)']
 SSH_PUBLIC_KEY=$(HOME)/.ssh/id_rsa.pub
+CONFIG_DIR=./config
+CLUSTER_CONFIG_TEMPLATE=manifest.template
+CLUSTER_CONFIG_PATH=$(CONFIG_DIR)/cluster-configuration/manifest.json
+DOMAIN:=redhat.com
```

Comment:
Let's use ?= for those variables, so they can be easily overriden ?
Same for CLUSTER_NAME and HOSTNAME